scheduler/src/cook/mesos/unscheduled.clj

```diff
@@ -160,14 +174,17 @@
     (case (:job/state job)
       :job.state/running [["The job is running now." {}]]
       :job.state/completed [["The job already completed." {}]]
-      (filter some?
-              [(check-exhausted-retries db job)
-               (check-exceeds-limit quota/get-quota
-                                    "The job would cause you to exceed resource quotas."
-                                    db job)
-               (check-exceeds-limit share/get-share
-                                    "The job would cause you to exceed resource shares."
-                                    db job)
-               (check-launch-rate-limit job)
-               (check-queue-position conn job)
-               (check-fenzo-placement conn job)]))))
+      (let [user (:job/user job)
+            pool-name (-> job :job/pool :pool/name)
+            running-jobs (get-jobs-by-user-and-state db user "running" 1000000 pool-name 30)]
```

Comment:
Nitpick: should we consider sending the `queued-jobs` explicitly too to defend against a performance regression in the future if more of the other `check-*` functions start requiring the waiting jobs?

Reply:
Done